packages/discovery-provider/src/queries/health_check.py

```diff
@@ -50,19 +50,21 @@ def health_check():
             "challenge_events_age_max_drift", type=int
         ),
         "plays_count_max_drift": request.args.get("plays_count_max_drift", type=int),
-        "reactions_max_indexing_drift": request.args.get(
-            "reactions_max_indexing_drift", type=int
+        "reward_manager_max_drift": request.args.get(
+            "reward_manager_max_drift", type=int, default=300
         ),
-        "reactions_max_last_reaction_drift": request.args.get(
-            "reactions_max_last_reaction_drift", type=int
+        "user_bank_max_drift": request.args.get(
+            "user_bank_max_drift", type=int, default=300
```

Comment:
300 seconds seems...high? We're allowing a 5 minute dropoff on indexing before we consider it unhealthy?

Reply:
It's def very high. Previously we weren't marking unhealthy by default _at all_ based on Solana indexers, so was being super relaxed to prevent noise. If we go five minutes without a succeed we know _for sure_ somethings completely busted and we should page/SDK should reselect etc. Healthz could set its own more narrow desired criteria via query params, like it does today with plays.

This could def be a conversation though - defaults are powerful and I'd be happy to make this like a single minute.

Reply:
I think as long as client isn't reselecting based off of this, it's probably ok to be 5 minutes

Reply:
sorry can you say more? like do you mean if the client SDK reselects are based on specific query params not the default? or that client doesn't reselect based on solana indexing at all?

Reply:
I would worry that an issue on the solana side would just halt all of audius and prevent even reads from working on discovery - but these aren't even incorporated into the sdk selection atm, so it's a non-concern.

all the more reason to get the "acdc/core" layer to validate TX instead of doing it here

Reply:
they are incorporated into SDK selection currently - the node gets marked as "behind" if the slot or time drifts are too far: https://github.com/AudiusProject/audius-protocol/blob/cc11317c5a3434bf77d4c218ca3463c4c9118368/packages/sdk/src/sdk/services/DiscoveryNodeSelector/healthChecks.ts#L57-L70

That doesn't preclude the SDK from selecting those nodes, it just tries to find a node that _isn't_ behind first.

The change here _does_ actually bubble up the unhealthy indexers to `discovery_provider_healthy` and to 500 - but I can revert that

Reply:
got it. i think that we will cause problems if we bubble that up atm, so i would say we should revert that